### DISCUSSION TITLE
Is it possible to use WindowScroller?

### DISCUSSION THREAD

**Discussion by akarabach** [Lodin/react-vtree#29] [Q&A] [answered]
Hi. I want to use `<WindowScroller />` or somehow implement window scroller for this tree ? Do you have an example of it or its impossible to use window scroller ?

**Comment by Lodin** [comment, marked as answer]
Hi @akarabach. Yes, I don't see any reason why you cannot. This library is just a thin wrapper over the `react-window`, so you if follow [this thread](https://github.com/bvaughn/react-window/issues/30) you may get it working. As I can see, there are some issues but it looks that users have solved them.
